ifrs17/Utils/Queries.ipynb

```diff
@@ -183,18 +183,18 @@
         {
             "cell_type": "code",
             "source": [
-                "public static async Task<Dictionary<string, YieldCurve>> LoadLockedInYieldCurveAsync(this IQuerySource querySource, Args args, ",
-                "\n                                                                                                       IEnumerable<DataNodeData> dataNodes)",
+                "public static async Task<Dictionary<string, YieldCurve>> LoadLockedInYieldCurveAsync(this IQuerySource querySource, Args args, IEnumerable<DataNodeData> dataNodes)",
+                "\n",
                 "\n{",
                 "\n    var lockedInYieldCurveByGoc = new Dictionary<string, YieldCurve>();",
                 "\n    foreach (var dn in dataNodes.Where(x => x.ValuationApproach == ValuationApproaches.BBA))",
                 "\n    {",
                 "\n        var argsNew = args with {Year = dn.Year, Month = dn.Month, Scenario = args.Scenario};",
```

Comment:
shouldn't this be monthInAYear instead of dn.Month?

Reply:
done